src/midi/live/view.rs

```diff
@@ -32,8 +32,8 @@ impl LiveNoteViewData {
             parser,
             columns,
             view_range: MIDIViewRange {
-                start: 0.0,
-                end: 0.0,
```

Comment:
Nitpick: is this needed? Infinities tend to have weird side effects and cause confusing bugs down the line

Reply:
Not sure if I can do this another way, but it can't be zero because then it will complain that I am trying to seek backwards at the beginning (since the initial value is 0 and I am requesting negative time).

Reply:
Huh ok, fair enough